docs/how-to/setup/set-up-snapcraft.rst

```diff
@@ -29,127 +29,127 @@ Install Snapcraft
 
     .. code:: bash
 
-      sudo snap install snapcraft --classic
-
+        snap install snapcraft --classic
 
   .. group-tab:: macOS
 
-    #. `Install Brew <https://brew.sh#install>`_.
-    #. Install Snapcraft:
+    First, `install Brew <https://brew.sh#install>`_.
+
+    Then, install Snapcraft through Brew:
 
-       .. code:: bash
+    .. code:: bash
 
         brew install snapcraft
 
   .. group-tab:: Windows
 
-    #. `Install WSL2 with Ubuntu 20.04 or higher
-       <https://learn.microsoft.com/en-us/windows/wsl/install#install-wsl-command>`_.
-    #. In WSL2, install Snapcraft:
+    First, `install WSL2 with Ubuntu 20.04 or higher
+    <https://learn.microsoft.com/en-us/windows/wsl/install#install-wsl-command>`_.
 
-       .. code:: bash
+    Then, install Snapcraft in WSL2:
 
-         sudo snap install snapcraft
+    .. code:: bash
+
+        snap install snapcraft --classic
 
 
 Install a build provider
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-Snapcraft is compatible with two build providers -- Multipass and LXD. It
-relies on a build provider to host an isolated build environment, like a
-sandbox. Inside this environment, software can be built and packaged as snaps
-without making potentially destructive changes to the host system.
+Snapcraft relies on a *build provider* to create an isolated build environment, like a
+sandbox. Inside this environment, software can be built and packaged as snaps without
+making potentially destructive changes to the host system.
 
-For **core22 and higher**, LXD is the default provider on Linux, and Multipass
-is the default on macOS and Windows.
+.. list-table::
+    :widths: 1 3 2
 
-For **core20**, Multipass is the default provider on all platforms.
+    * - Build provider
+      - Description
+      - Default on
+    * - `LXD <https://canonical.com/lxd>`_
+      - Creates and manages Linux container images. It can operate inside VMs.
+      - Snapcraft 7 on Linux
+    * - `Multipass <https://multipass.run>`_
+      - Creates and manages virtual machine (VM) build instances. It automates setup and
+        teardown of cloud-style Ubuntu VMs. It can't be used  reliably on platforms that
+        don't support nested virtualization. For instance, it most likely won't itself
+        run inside a VM.
+      - Snapcraft 6 on Linux
 
+        macOS and Windows
 
-Choose a build provider
-^^^^^^^^^^^^^^^^^^^^^^^
+When you first run Snapcraft, it installs the default build provider. If the default
+build provider isn't fit for purpose, you can install the alternative and then
```

Comment:
```suggestion
build provider isn't fit for your purpose, you can install the alternative and then
```

Reply:
"fit for purpose" is a fixed phrase in English, but it's maybe too fancy. [0c39d3c](https://github.com/canonical/snapcraft/pull/5537/commits/0c39d3c4bd696fd5b1c0e10dceae39fae22e58c7)

Reply:
Ha, certainly too fancy for me to recognize!